script.js

```diff
@@ -0,0 +1,68 @@
+const container= document.querySelector(".container");
+const count = document.querySelector("#count");
+const amount =document.getElementById("amount");
+const movie= document.getElementById("movie");
+const seats= document.querySelectorAll(".seat:not(.reserved)");
+getFromLocalStorage()
+calculateTotal();
+container.addEventListener("click", function(e){
+    if(e.target.classList.contains("seat")&& !e.target.classList.contains("reserved")){
+        e.target.classList.toggle("selected");
+        calculateTotal();
+
+    }
+});
+
+movie.addEventListener("change",function(e){
+    calculateTotal();
+})
+
+
+function calculateTotal(){
+    const selectedSeats= container.querySelectorAll(".seat.selected");
+
+    const selectedSeatsArr= [];
```

Comment:
bu iki arraye gerek var mi tam anlayamadim. Hala github ile ilgili sorun var bilgisayarimda o yüzden indirip inceleyemedim. Ama bir bak bu arrayler olmadan olmuyor mu buradaki islem?

Reply:
Burada yapılmak istenen koltukların index lerini tespit edebilmek. Bunun için map methodu kullanılıyor. Map methodu da array üzerinde çalıştığı için mecburen bunlar oluşturuldu.